common/install_base.sh

```diff
@@ -67,3 +67,25 @@ fi
 # Cleanup package manager
 apt-get autoclean && apt-get clean
 rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install OpenJDK
+# https://hub.docker.com/r/picoded/ubuntu-openjdk-8-jdk/dockerfile/
+
+sudo apt-get update && \
+	apt-get install -y openjdk-8-jdk && \
```

Comment:
Hmm, Java is pretty big, I'm not sure we should always install it on every Docker image

Reply:
Oh, but that's exactly what this PR is about. okey dokey, I guess you have a reason for it? :>